client/my-sites/site-settings/site-tools/link.jsx

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 
 const noop = () => {};
 
-const SiteToolsLink = ( { description, href, isWarning, onClick, title } ) => {
+const SiteToolsLink = ( { description, href, isWarning, onClick = noop, title } ) => {
```

Comment:
It's not 100% straightforward, because `onClick` prop triggers addition of `is-clickable` class to `<Card>`, leading to `cursor: pointer` style, but I verified that for `SiteToolsLink` this is OK because all of them are `a href` links that are "clickable" by default.